Components.Tests/RenderTests/RenderTests.cs

```diff
@@ -110,6 +110,23 @@ public void RenderSessionTimeoutControl(Model sut)
             result.ToString().Should().ContainAll("class='wb-sessto'", "inactivity", "reactionTime", "sessionalive", "logouturl", "refreshCallbackUrl", "refreshOnClick", "refreshLimit", "method", "additionalData");
         }
 
+        [Theory, AutoNSubstituteData]
```

Comment:
Should CHANGELOG be update to mention session timeout?

Reply:
done